express-notes-api/index.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-console */
+const json = require('./data.json');
+const fs = require('fs');
+
+const express = require('express');
+
+const app = express();
+
+app.use(express.json());
+
+const notes = [];
+
+app.get('/api/notes', function (req, res) {
+  if (json.notes !== {}) {
```

Comment:
This condition will never be true, you can skip it. Objects never equal each other, even if they have the same properties.